src/runAll.js

```diff
@@ -32,14 +31,16 @@ module.exports = async function runAll(config) {
 
   debug('Resolved git directory to be `%s`', gitDir)
 
-  sgf.cwd = gitDir
-
   /* files is an Object{ filename: String, status: String } */
-  const files = await pify(sgf)('ACM')
-  const filenames = files.map(file => file.filename)
-  debug('Loaded list of staged files in git:\n%O', filenames)
+  const files = await getStagedFiles()
+
+  if (!files || !Array.isArray(files)) {
```

Comment:
This should be gone after we throw errors in the function. Instead let’s wrap it into try catch?

Reply:
See https://github.com/okonet/lint-staged/pull/624#discussion_r294611865